eng/pipelines/templates/stages/vmr-build.yml

```
@@ -221,12 +221,12 @@ stages:
 
     - template: ../jobs/vmr-build.yml
       parameters:
-        buildName: Debian11_Offline_MsftSdk
+        buildName: Ubuntu2004_Offline_MsftSdk
```

Fixing the version. Also think it would be good to specify `arm64` in the name since that's what makes this leg special compared to the others.

```suggestion
        buildName: Ubuntu2204Arm64_Offline_MsftSdk
```

---

Thanks for noticing the version + pointing out the architecture part, will fix